### DISCUSSION TITLE
为什么提交到bing时总是报错

### DISCUSSION THREAD

**Discussion by xieyanshe666** [cjh0613/hexo-submit-urls-to-search-engine#16] [Q&A]
Bing response:  [90mundefined[39m
Bing response:  [90mundefined[39m
Bing response:  [90mundefined[39m
Bing response:  [90mundefined[39m
Bing response:  [90mundefined[39m
Bing response:  [90mundefined[39m
Bing response:  [90mundefined[39m
Bing response:  [90mundefined[39m
Bing response:  [90mundefined[39m
Bing response:  [90mundefined[39m
Bing response:  [90mundefined[39m
Bing response:  [90mundefined[39m
Bing response:  [90mundefined[39m
Bing response:  [90mundefined[39m

总是出现这种错误，不知道为什么？

**Comment by gitlGl** [comment]
同这个问题，你解决了吗？

**Reply by xieyanshe666** [reply]
没啊
